api/db/changelog/db.changelog-101-billing-project-garbage-collection.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+  xmlns="http://www.liquibase.org/xml/ns/dbchangelog/1.9"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog/1.9
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-1.9.xsd">
+  <changeSet author="thibault" id="changelog-101-billing-project-garbage-collection">
+
+    <createTable tableName="billing_project_garbage_collection">
```

Comment:
does mysql let you have column names longer than 31 characters?

Reply:
Looks like 64: https://dev.mysql.com/doc/refman/8.0/en/identifier-length.html